examples/helper/avian.rs

```diff
@@ -0,0 +1,54 @@
+use avian2d::prelude::*;
+use bevy::prelude::*;
+
+const MOVE_SPEED: f32 = 200.;
+const GRAVITY_SCALE: f32 = 10.0;
+
+#[derive(Default, Clone, Component)]
+pub struct PlayerMarker;
+
+#[allow(unused)]
+pub fn spawn_player(commands: &mut Commands, radius: f32, spawn_position: Vec2) {
+    commands
+        .spawn(RigidBody::Dynamic)
+        .insert(PlayerMarker)
+        .insert(Name::new("PlayerControlledObject (Avian2D physics)"))
+        .insert(Collider::circle(radius))
+        .insert(GravityScale(GRAVITY_SCALE))
```

Comment:
I'm not sure we actually have any gravity in the example using the controller ?

Reply:
In avian I had to add the gravity resource otherwise there is no gravity by default. Fixed it.